app/src/graphql/operations.graphql

```diff
@@ -35,7 +36,24 @@ subscription CurrentGame($joinCode: String!) {
       id
       value
     }
-    cards {
+    cards(
+      where: {
+        _or: [
+          {
+            _and: [
+              { game: { screen_cards: { _eq: true } } }
+              { is_allowed: { _eq: true } }
```

Comment:
i could see this being renamed to is_screened_out, to match the screen_cards column

Reply:
Yeah, I like that thought.

Reply:
@namoscato let's get it all working and i can do the rename in one go.